prog/postgres/postgres_server_nexus.rb

```diff
@@ -272,6 +273,10 @@ def before_run
       hop_configure
     end
 
+    when_restart_set? do
+      push self.class, frame, "restart"
```

Comment:
when you push it, and do not handle the `destroy` scenario, we may end-up with a strand without model, remember? :) We need to solve that issue soon, both for this and Minio

Reply:
Added that as a separate commit.